homeassistant/components/neato.py

```diff
@@ -0,0 +1,82 @@
+"""
+Support for Neato botvac connected vacuum cleaners.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/neato/
+"""
+import logging
+from datetime import timedelta
+from urllib.error import HTTPError
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.helpers import discovery
+from homeassistant.util import Throttle
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['https://github.com/jabesq/pybotvac/archive/v0.0.1.zip'
+                '#pybotvac==0.0.1']
+
+DOMAIN = 'neato'
+HUB = None
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
+    """Setup the Verisure component."""
+    from pybotvac import Account
+
+    global HUB
```

Comment:
Please don't use globals but instead use `hass.data`, a dictionary that can contain component data.